portal-api/src/v1/users/sign-in-link.service.js

```diff
@@ -0,0 +1,69 @@
+const { ObjectId } = require('mongodb');
+const crypto = require('node:crypto');
+const sendEmail = require('../email');
+const { EMAIL_TEMPLATE_IDS, SIGN_IN_LINK_EXPIRY_MINUTES } = require('../../constants');
+const db = require('../../drivers/db-client');
+
+module.exports.createAndEmailSignInLink = async (user) => {
+  const {
+    _id: userId,
+    email: userEmail,
+    firstname: userFirstName,
+    surname: userLastName,
+  } = user;
+
+  const signInCode = await createSignInCode();
+  await saveSignInCodeHashAndSalt({ userId, signInCode });
+  return sendSignInLinkEmail({
+    userEmail,
+    userFirstName,
+    userLastName,
+    // TODO DTFS2-6750: introduce an environment variable to replace localhost
```

Comment:
Sorry to be a pain -- do we want to change this todo to 6680 to ensure we dont miss it (if were quite happy it wont be missed, im happy)?

Reply:
I'll raise a ticket for it now and remove the TODO